prow/cmd/deck/main.go

```diff
@@ -157,16 +156,16 @@ func httpChunking(log io.ReadCloser, w http.ResponseWriter) {
 	if !ok {
 		logrus.Warning("Error getting flusher.")
 	}
-	cw := httputil.NewChunkedWriter(w)
 	reader := bufio.NewReader(log)
 	for {
 		line, err := reader.ReadBytes('\n')
 		if err != nil {
-			if err == io.EOF {
-				break
-			}
+			// TODO(rmmh): The log stops streaming after 30s.
```

Comment:
May it be related to a loadbalancer timeout? I am hitting this on GCE.

Reply:
Interesting. @rmmh plan on investigating further?

Reply:
xref: https://github.com/openshift/release/issues/3